webpack.config.js

```diff
@@ -35,95 +34,56 @@ const LIBRARY_BUNDLE_CONFIG = env => ({
 
   output: {
     // Generate the bundle in dist folder
-    path: resolve('./dist'),
-    filename: 'index.js',
-    library: 'kepler.gl'
-    // libraryTarget: 'umd'
+    path: resolve('./umd'),
+    filename: 'keplergl.js',
+    globalObject: 'this',
+    library: '[name]',
+    libraryTarget: 'umd'
   },
 
-  // Exclude any non-relative imports from resulting bundle
-  externals: [
-    /^[a-z\.\-0-9]+$/
-  ],
-
-  module: {
-    rules: [{
-      test: /\.js$/,
-      loader: 'babel-loader',
-      include: [SRC_DIR]
-    }, {
-      test: /\.json$/,
-      loader: 'json-loader'
-    }, {
-      test: /\.scss$/,
-      use: ['style-loader', 'css-loader', 'sass-loader'],
-      include: [SRC_DIR]
-    }]
-  },
-
-  node: {
-    fs: 'empty'
-  },
-
-  plugins: [
-    // leave minification to app
-    // new webpack.optimize.UglifyJsPlugin({comments: false}),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify(env),
-        BROWSER: JSON.stringify(true),
-      }
-    })
-  ]
-});
-
-const TEST_BROWSER_CONFIG = {
-  devServer: {
-    stats: {
-      warnings: false
+  // let's put everything in
+  externals: {
+    react: {
+      root: 'React',
+      commonjs2: 'react',
+      commonjs: 'react',
+      amd: 'react',
+      umd: 'react'
     },
-    quiet: true
-  },
-
-  // Bundle the tests for running in the browser
-  entry: {
-    'test-browser': resolve('./test/browser.js')
-  },
-
-  // Generate a bundle in dist folder
-  output: {
-    path: resolve('./dist'),
-    filename: '[name]-bundle.js'
-  },
-
-  devtool: '#inline-source-maps',
-
-  resolve: {
-    alias: {
-      'kepler.gl/test': resolve('./test'),
-      'kepler.gl': resolve('./src')
+    'react-dom': {
+      root: 'ReactDOM',
+      commonjs2: 'react-dom',
+      commonjs: 'react-dom',
+      amd: 'react-dom',
+      umd: 'react-dom'
     }
   },
 
   module: {
-    rules: []
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        include: [SRC_DIR]
+      },
+      {
+        test: /\.json$/,
+        loader: 'json-loader',
+        type: 'javascript/auto'
+      },
+      {
+        test: /\.scss$/,
```

Comment:
Is anything using Sass in kepler/example?

Reply:
left over